astropy/utils/iers/iers.py

```diff
@@ -1021,7 +1024,7 @@ def auto_open(cls, files=None):
             if self is None or trial.expires > self.expires:
                 self = trial
                 self.meta['data_url'] = str(f)
-                if self.expires > good_enough:
+                if conf.auto_max_age is not None and self.expires > good_enough:
```

Comment:
Keep this as is to still pick the best file, and then at line 1034 only warn if the max age is set:
```
if conf.auto_max_age is not None and self.expires < self._today():
    warn(...)
```

Reply:
Done. Thanks!